connector/ws/src/lib.rs

```diff
@@ -61,6 +59,7 @@ struct Inner<S: connector::Supervisor> {
 
 impl<S: connector::Supervisor> WsConnector<S> {
     pub fn new(supervisor_id: Uuid, config: WsConnectorConfig, supervisor: Weak<S>) -> Self {
+        tracing::error!("UH OH");
```

Comment:
Hm? :laughing:

Reply:
Ah, extra debugging statement that I forgot to remove.